File: Sample/Sample/MessageCenterViewController.m
/* Copyright Airship and Contributors */

#import "MessageCenterViewController.h"

@implementation MessageCenterViewController

- (void)viewWillAppear:(BOOL)animated {
    [super viewWillAppear:animated];
    self.messageCenterStyle = [UAMessageCenterStyle styleWithContentsOfFile:@"MessageCenterStyle"];
    
    self.messageViewController.extendedLayoutIncludesOpaqueBars = YES;
    self.listViewController.extendedLayoutIncludesOpaqueBars = YES;

}

- (void)showInbox {
    [self.listViewController.navigationController popToRootViewControllerAnimated:YES];
}

@end

File: SwiftSample/SwiftSample/MessageCenterViewController.swift
/* Copyright Airship and Contributors */

import Foundation
import UIKit
import AirshipCore
import AirshipMessageCenter

class MessageCenterViewController : DefaultMessageCenterSplitViewController {

    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        self.messageCenterStyle = MessageCenterStyle(contentsOfFile: "MessageCenterStyle")
        
        self.messageViewController.extendedLayoutIncludesOpaqueBars = true
        self.listViewController.extendedLayoutIncludesOpaqueBars = true
    }
    
    func showInbox() {
        self.listViewController.navigationController?.popToRootViewController(animated: true)
    }
}
